test/unit/app/browser/reducers/sitesReducerTest.js

```diff
@@ -149,8 +149,8 @@ describe('sitesReducerTest', function () {
       moveAction.prepend = true
       newState = sitesReducer(Immutable.fromJS(newState), moveAction).toJS()
       assert.equal(Object.keys(newState.sites).length, 3)
-      assert.equal(Object.values(newState.sites)[2].location, url)
-      assert.equal(Object.values(newState.sites)[2].order, 1)
+      assert.equal(Object.values(newState.sites)[1].location, url)
```

Comment:
Is this correct change, because the comment at the top states that site should be at the third position (2), but you changed this value to 1